manage.py

```diff
@@ -447,3 +446,116 @@ def update_qnr_authored(qnr_id, authored, actor):
         user_id=acting_user.id,
         subject_id=qnr.subject_id)
     print(message)
+
+
+@click.option('--src_id', help="Source Patient ID (WILL BE DELETED!)")
+@click.option('--tgt_id', help="Target Patient ID")
+@click.option(
+    '--actor',
+    help='email address of user taking this action, for audit trail'
+)
+@app.cli.command()
+def merge_users(src_id, tgt_id, actor):
+    """Copy useful portion of source to target user and delete source"""
+    from flask import current_app
+    from portal.models.audit import Audit
+    from portal.models.user import internal_identifier_systems
+    from portal.models.tou import ToU
+
+    try:
+        acting_user = User.query.filter(
+            func.lower(User.email) == actor.lower()).one()
+    except NoResultFound:
+        raise ValueError("email for acting user <{}> not found".format(actor))
+
+    acting_user.check_role(permission='edit', other_id=src_id)
+    acting_user.check_role(permission='edit', other_id=tgt_id)
+
+    src_user = User.query.get(src_id)
+    tgt_user = User.query.get(tgt_id)
+
+    if not all((src_user, tgt_user)) or (
+            src_user.birthdate != tgt_user.birthdate):
+        raise ValueError("Birth dates don't match; can't continue")
+    if src_user.auth_providers.count() > 0:
+        raise ValueError("extend to include auth_providers")
+
+    if src_user.identifiers != tgt_user.identifiers and (
+            click.confirm("Add identifiers \n\t{} \nto \n\t{}".format(
```

Comment:
You may be able to fold some of these confirmations into decorators ([using `prompt=True`](http://click.palletsprojects.com/en/5.x/options/#option-prompting)), but would likely lose the granularity